### ISSUE TITLE
uncertainty quantification of a neural network 

### ISSUE THREAD

**Issue by admodadmod** [uncertainty-toolbox/uncertainty-toolbox#41]
Hi,
I have a trained neural network. How can I used the Uncertainty Toolbox to quantify uncertainty of the neural network.
How should I calculate predictions_std (a vector of standard deviations)?

**Comment by sujanme25** [comment]
Hi,
I am stuck in recalibration, I got the mean and SD prediction from DNN and got the average caliibration using this toolbox, Now what i have to do to perform the recalibration. Do i need to set aside validation data to further recalibrate?

**Comment by IanChar** [comment]
Hello you do need to set aside another validation set in order to recalibrate. I would look at the [toolbox tutorial](https://uncertainty-toolbox.github.io/tutorial/) if you haven't already. The end of the tutorial has an example of recalibration for this case.
